eng/common/internal-feed-operations.ps1

```diff
@@ -17,48 +17,48 @@ Set-StrictMode -Version 2.0
 function SetupCredProvider {
```

Comment:
Aren't all files under eng/common automatically propagated from dotnet/arcade?

It also looks like all of the changes here are whitespace.

If that's true, let's revert all these eng/common changes.

Reply:
you are right, The editor accidentally trim line ends. I will revert it